packages/docs/src/components/DocsCode.vue

```diff
@@ -1,6 +1,16 @@
 <template>
+  <va-tabs v-if="!isString" v-model="index">
+    <template #tabs>
+      <va-tab
+        v-for="tab in tabs"
+        :key="tab"
+      >
+        {{ tab }}
+      </va-tab>
+    </template>
+  </va-tabs>
   <prism-wrapper
-    :code="formattedCode"
+    :code="contents[index-1]"
```

Comment:
```suggestion
    :code="contents[index]"
```

Reply:
`model:value` of `va-tabs` start with 1, but contents start with 0, I have deliberately checked this out in demo book that start with `yarn serve`, so I think `index` should probably stay the same.

Reply:
I guess there is a bug in our `VaTabs` component with tab indexes. So, I opened a PR #1702 . Let's wait for his approval.

Reply:
Sure, thank you @RVitaly1978.

Reply:
PR https://github.com/epicmaxco/vuestic-ui/pull/1702 has been approved and merged into the develop branch. So now `model:value` of `va-tabs` is equal to its index. You can apply this suggestion)